ARDemos/ARDemos/ViewController.swift

```diff
@@ -0,0 +1,180 @@
+import UIKit
+import ARKit
+import SceneKit
+
+class ViewController: UIViewController {
+
+    @IBOutlet var sceneView: ARView!
+    @IBOutlet var bottomBar: BottomBar!
+    
+    private var responseData: ResponseData?
+    private var currentModel: Model?
+    private var modelNodeModel: SCNNode?
+    private var planeNodeModel: SCNNode?
+    private var lightNodeModel: SCNNode?
+    
+    private let modelFactory = DataFactory()
+    private let jsonFileName = "ModelsData"
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        // Set the view's delegate
```

Comment:
comments if function's body are a strong indicator that you can refactor this logic with a new method with a descriptive name. I can see there are other examples in this class as well

Reply:
This comments are AR specific and come from the AR template that we started defined by Apple. I though about removing them but since this is an AR demo I saw some value on having them on the project alos